### PR TITLE
docs: multi project

### PR DESCRIPTION
**Pull request by jon-stewart**
## Summary

Examples for multi project creation.  Referencing https://docs.lacework.com/onboarding/gcp-compliance-and-audit-log-integration-terraform-using-google-cloud-shell#configure-multiple-projects-with-terraform

## How did you test this change?

terraform plan

## Issue

https://lacework.atlassian.net/browse/GROW-1502

### CODE REVIEW

File: examples/project-level-audit-log-multi/README.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+  for_each   = var.projects
+  project_id = each.key
+}
+```
+
+Run Terraform:
+```
+$ terraform init
+$ GOOGLE_CREDENTIALS=account.json GOOGLE_PROJECT=my-project terraform apply
```
Comment (afiune) [inline examples/project-level-audit-log-multi/README.md:38]: The project is already set in the provider block
```suggestion
$ GOOGLE_CREDENTIALS=account.json terraform apply
```

File: examples/project-level-audit-log-multi/README.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  description = "Map of project configuration with Lacework."
+  type        = map
+  default     = {
+    project-id-1 = "first project",
+    project-id-2 = "second project"
+  }
+}
+
+module "gcp_audit_log" {
+  source = "../../"
```
Comment (afiune) [inline examples/project-level-audit-log-multi/README.md:28]: Use real source information for documentation:

```suggestion
  source = "lacework/audit-log/gcp"
  version = "~> 3.0"
```

File: examples/project-level-audit-log-multi/README.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
+# Integrate Multiple GCP Project with Lacework
+The following provides an example of integrating multiple Google Cloud Project with Lacework for Cloud Audit Log analysis.
+
+The fields required for this example are:
+
+| Name       | Description                                                                                 | Type     |
+|------------|---------------------------------------------------------------------------------------------|----------|
+| `projects` | The id of each Project, which will be used to deploy required resources for the integration | `map` |
```
Comment (afiune) [inline examples/project-level-audit-log-multi/README.md:8]:
```suggestion
| `projects` | Map of projects which will be used to deploy required resources for each integrations | `map` |
```

File: examples/project-level-audit-log-multi/versions.tf [hunk 4]
```diff
@@ -0,0 +1,11 @@
+# required for Terraform 13
+terraform {
+  required_providers {
+    google = "4.36.0"
```
Comment (afiune) [inline examples/project-level-audit-log-multi/versions.tf:4]: We recommend pessimistic version constraint
```suggestion
    google = "~> 4.36"
```

File: examples/project-level-audit-log-multi/README.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
+# Integrate Multiple GCP Project with Lacework
+The following provides an example of integrating multiple Google Cloud Project with Lacework for Cloud Audit Log analysis.
```
Comment (lacework-aaronscheiber) [inline examples/project-level-audit-log-multi/README.md:2]:
```suggestion
The following provides an example of integrating multiple Google Cloud projects with Lacework for cloud Audit Log analysis.
```

File: examples/project-level-audit-log-multi/README.md [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+
+The fields required for this example are:
+
+| Name       | Description                                                                                 | Type     |
+|------------|---------------------------------------------------------------------------------------------|----------|
+| `projects` | Map of projects which will be used to deploy required resources for each integrations | `map` |
```
Comment (lacework-aaronscheiber) [inline examples/project-level-audit-log-multi/README.md:8]:
```suggestion
| `projects` | Map of projects that will be used to deploy required resources for each integration | `map` |
```

File: examples/project-level-audit-log-multi/README.md [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+}
+```
+
+Run Terraform:
+```
+$ terraform init
+$ GOOGLE_CREDENTIALS=account.json terraform apply
+```
+
+For detailed information on integrating Lacework with Google Cloud see [GCP Compliance and Audit Trail Integration - Configure Multiple Porjects with Terraform](https://docs.lacework.com/onboarding/gcp-compliance-and-audit-log-integration-terraform-using-google-cloud-shell#configure-multiple-projects-with-terraform)
```
Comment (lacework-aaronscheiber) [inline examples/project-level-audit-log-multi/README.md:42]:
```suggestion
For detailed information on integrating Lacework with Google Cloud see [GCP Compliance and Audit Trail Integration - Configure Multiple Projects with Terraform](https://docs.lacework.com/onboarding/gcp-compliance-and-audit-log-integration-terraform-using-google-cloud-shell#configure-multiple-projects-with-terraform)
```

**Review by lacework-aaronscheiber — CHANGES_REQUESTED** [review]

File: examples/project-level-audit-log-multi/README.md [hunk 8]
```diff
@@ -0,0 +1,42 @@
+# Integrate Multiple GCP Project with Lacework
```
Comment (lacework-aaronscheiber) [inline examples/project-level-audit-log-multi/README.md:1]:
```suggestion
# Integrate Multiple GCP Projects with Lacework
```

**Review by lacework-aaronscheiber — DISMISSED** [review]
Additional changes

**Review by afiune — APPROVED** [review]

**Pull request merged**